packages/code-sandbox/src/callbackLogFormatter.ts

```diff
@@ -0,0 +1,20 @@
+import { formatCode } from '@stardust-ui/docs-components'
+import * as React from 'react'
+
+const callbackLogFormatter = (props: string[]) => (
+  name: string,
+  e: React.SyntheticEvent,
+  data: Object,
+) => {
+  const pickedProps = props.reduce((acc, propName) => {
+    acc[propName] = data[propName]
+    return acc
+  }, {})
+
+  return [
+    `/*[${new Date().toLocaleTimeString()}]*/`,
```

Comment:
Not sure why this is in comment... It's not like the rest of the information is a running code...

Reply:
Plus, if we log something different then callback, it would be different then the default formatter styles...

Reply:
In the it will be highlighted by `Prism.JS` as Javascript code, without comment it looks ugly 😭 

![image](https://user-images.githubusercontent.com/14183168/64715770-5d439e80-d4c1-11e9-8e88-1ca2e2ecb861.png)

Reply:
Ok, let's then change the default one as well... Maybe that would be better